client.lua

```diff
@@ -194,32 +207,146 @@ local function StartFueling(vehicle)
     notify(fuel)
 end
 
-RegisterCommand('startfueling', function()
+local function StartFuelingWithCan(vehicle)
```

Comment:
Do we really need a separate function for this that does the exact same thing as the other one? Couple of ifs should do fine